include/basic_types.h

```diff
@@ -14,7 +14,15 @@
  *****************************************************************************/
 #ifndef __BASIC_TYPES_H__
 #define __BASIC_TYPES_H__
+#include <linux/version.h>
 
+#if (LINUX_VERSION_CODE < KERNEL_VERSION(5, 4, 0))
+#if __has_attribute(__fallthrough__)
+# define fallthrough                    __attribute__((__fallthrough__))
+#else
+# define fallthrough                    do {} while (0)  /* fallthrough */
+#endif
+#endif
```

Comment:
Not super sure where is best for this. Only here because I know it is included just about everywhere else